packages/legend-studio/src/stores/editor-state/element-editor-state/connection/ConnectionEditorState.ts

```diff
@@ -178,6 +184,12 @@ export class RelationalDatabaseConnectionValueState extends ConnectionValueState
         );
         return;
       }
+      case CORE_DATASOURCE_SPEC_TYPE.REDSHIFT: {
+        this.connection.setDatasourceSpecification(
+          new RedshiftDatasourceSpecification('', '', 5439),
```

Comment:
can we default to port 80 to make it consistent with the other default port numbers, or is there a reason we default to 5439?

Reply:
I used 5439 because that is the port that Redshift uses by default